src/Command/RoutesCommand.php

```diff
@@ -74,6 +80,32 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
         $io->helper('table')->output($output);
         $io->out();
 
+        $duplicateRoutes = [];
+
+        // Check duplicate routes
+        foreach (Router::routes() as $route) {
+            if ($duplicateRoutesCounter[$route->template] > 1) {
+                $methods = $route->defaults['_method'] ?? '';
+
+                $duplicateRoutes[] = [
+                    $route->options['_name'] ?? $route->getName(),
+                    $route->template,
+                    $route->defaults['plugin'] ?? '',
+                    $route->defaults['prefix'] ?? '',
+                    $route->defaults['controller'] ?? '',
+                    $route->defaults['action'] ?? '',
+                    is_string($methods) ? $methods : implode(', ', $route->defaults['_method']),
```

Comment:
```suggestion
                    is_string($methods) ? $methods : implode(', ', $methods),
```

Reply:
thx 😄 I changed it up also in the upper section of the command logic where the same change needed to be done.